src/components/new-request/search.vue

```diff
@@ -14,28 +14,32 @@
           label="Select an Action"
           :error-messages="getErrors.includes('request_action_cd') ? 'Please select an action' : ''"
           :items="requestActions"
+          item-value="[group,value]"
```

Comment:
It worked the first time!

![image](https://github.com/bcgov/namerequest/assets/10002889/5a258cd0-f016-4fd5-978d-e84f95275c79)

Ref: https://v2.vuetifyjs.com/en/api/v-select/#props-item-value

Reply:
Why are you adding this prop exactly Sev?

Reply:
To fix a funny behaviour that Yui found and reported in the ticket: if you select the first "NEW" action in the list (in first group), and then you look at the second group, its "NEW" action also appears selected.

This prop, which works along with the `items` array prop, tells the v-select which "value" to keep track of. So I created this compound value which is now unique, and the v-select now knows the difference between the first (`[0,NEW]`) and the second (`[1,NEW]`) items.

Re:
![image](https://github.com/bcgov/namerequest/assets/10002889/f0186ab8-ba60-4938-8589-ce06701227c2)

Reply:
Oh, OK. I didn't notice that while using the component in my PR. Thanks for explaining Sev. Noted.

Reply:
I didn't notice either! But that Yui feller is pretty sharp 😉